VMAccess/README.md

```diff
@@ -1,20 +1,22 @@
 # VMAccess Extension
```

Comment:
You should work with Gabby to reflect the changes you made here in the Microsoft docs here https://learn.microsoft.com/en-us/azure/virtual-machines/extensions/vmaccess